content/en/platform/corda/4.8/open-source/key-concepts-ecosystem.md

```diff
@@ -12,59 +12,40 @@ menu:
-
-All of the nodes in the network have the *potential* to communicate with other nodes.
-
-**Why do we say “potential” to communicate?**
+Corda is different from other distributed ledgers because all communication between nodes is point-to-point, and only shared on a need-to-know basis. It's also encrypted using transport-layer security. There are **no global broadcasts** to all parties on a network, but all of the nodes in a network can send messages directly to other nodes. If the recipient is offline, the message waits in an outbound queue until they are online again—just like email.
 
-Because the connections on the graph do not have to be persistent. On the networking level, Corda uses persistent queues, but, as with email, if your recipient is offline, your messages will wait on an outbound queue until the recipient comes online.
+## How do I join a network?
 
-## Identities and Discovery
+Unlike traditional blockchain, Corda networks are semi-private. To join a network, you must obtain a certificate from the network operator. This
+certificate maps the node's identity on Corda to a real-world legal identity and a [public key](https://www.investopedia.com/terms/p/public-key.asp).
 
-Each node has a single well-known identity. The node’s identity is used to represent the node in transactions; for example, if the node were involved in a transaction to purchase an asset.
+The network operator enforces rules that stipulate what information nodes must provide and the know-your-customer (KYC) processes they must undergo before being granted this certificate.
 
-The network map service maps each well-known node identity to an IP address. These IP
-addresses are used for messaging between nodes.
+## How are nodes identified on a network?
+The network map service maps each node identity to an IP address. Nodes use these IP addresses to send messages to each other.
 
 Nodes can also generate confidential identities for individual transactions. The certificate chain linking a
-confidential identity to a well-known node identity or real-world legal identity is only distributed on a need-to-know
-basis. If confidential identities are being used, this ensures that even if an attacker gets access to an unencrypted transaction, they cannot identify the
-transaction’s participants without additional information.
-
-**How do Corda nodes Discover each other?**
+confidential identity to a node identity or real-world legal identity is only distributed on a need-to-know
+basis. Nodes can use confidential identities to protect themselves in the event that an attacker gets access to an unencrypted transaction, they cannot identify the participants without additional information.
 
+## How do nodes find each other on a network?
 Corda nodes discover each other via a **network map service**. You can think of this service as a phone book, which publishes a list of peer nodes that includes metadata about who they are and what services they can offer.
```

Comment:
```suggestion
Corda nodes discover each other via a *network map service*. You can think of this service as a phone book, which publishes a list of peer nodes that includes metadata about who they are and what services they can offer.
```
I'd introduce a new concept using italics. Not sure if our style guide says anything about this.

Reply:
agreed! good one